doc/topics/cloud/vmware.rst

```diff
@@ -146,10 +146,7 @@ Set up an initial profile at ``/etc/salt/cloud.profiles`` or
             name: 10.40.50-600-Prod
             adapter_type: vmxnet3
             switch_type: distributed
-            ip: 10.40.50.123
-            gateway: [10.40.50.110]
-            subnet_mask: 255.255.255.128
-            domain: example.com
```

Comment:
I was simply trying to illustrate that you don't have to set an IP address. Would it be a good idea to restore those lines and move `mac: ` line to up to `Network adapter 2`?